src/token_list.py

```diff
@@ -4,7 +4,7 @@
 import requests
 
 # pylint: disable=line-too-long
-ALLOWED_TOKEN_LIST_URL = "https://raw.githubusercontent.com/gnosis/cow-dex-solver/main/data/token_list_for_buffer_trading.json"
+ALLOWED_TOKEN_LIST_URL = "http://token-list.cow.eth.link/"
```

Comment:
If it is not reliably fetchable, then I would not want to depend on it.

I would either update the file in the gnosis repo or make sure that cowswap is hosting it somewhere.

Reply:
Sounds very practical!

Reply:
However, we must ensure to keep this list updated with the primary source of truth. How can we ensure this?

Reply:
You can have a unit test that verifies the ENS record matches what you expect.

Reply:
I am referring more to the fact that Alex is suggesting not using this url and stick with the old one, but updating the content at the current url.

Reply:
I just don't think it is so practical to continue using the current URL, because I don't want to have to be the one to manually update an outdated file every week.

Reply:
Right, my point is that you can ensure that the file is up to date (regardless of where you store it) by checking the ENS record. Just trying to answer:

> we must ensure to keep this list updated with the primary source of truth. How can we ensure this?

Reply:
Ah yes I see what you mean now. Thanks.